eng/pipelines/templates/stages/archetype-java-release-batch.yml

```diff
@@ -0,0 +1,502 @@
+                        parameters:
+                          ArtifactName: ${{parameters.ArtifactName}}-${{artifact.name}}-javadevfeed-$(System.JobAttempt)
+                          ArtifactPath: $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed
+
+        - deployment: UpdatePackageVersion
+          displayName: "Update Package Version"
+          condition: and(succeeded(), ne(variables['Skip.UpdatePackageVersion'], 'true'))
+          environment: github
+          dependsOn: PublishPackage
+          variables:
+            - template: /eng/pipelines/templates/variables/globals.yml
+          pool:
+            name: azsdk-pool-mms-win-2019-general
+            vmImage: MMS2019
+          strategy:
+            runOnce:
+              deploy:
+                steps:
+                  - download: none
+
+                  - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+                    parameters:
+                      Paths:
+                        - '${{ parameters.ServiceDirectory }}'
+                        - '**/*.xml'
+                        - '!sdk/**/test-recordings'
+                        - '!sdk/**/session-records'
+
+                  - task: UsePythonVersion@0
+                    displayName: 'Use Python 3.6'
+                    inputs:
+                      versionSpec: '3.6'
+
+                  # Apply the version increment to each library, which only updates the version_client.txt file,
+                  # and then call to update_versions will then apply all of the updates.
+                  - ${{ each artifact in parameters.Artifacts }}:
+                    - script: |
+                        python --version
+                        python eng/versioning/set_versions.py --build-type ${{ parameters.SDKType }} --increment-version --ai ${{ artifact.name }} --gi ${{ artifact.groupId }}
+                      displayName: Increment package version for ${{ artifact.groupId }} ${{ artifact.name }}
+
+                  - script: |
+                      python --version
+                      python eng/versioning/update_versions.py --update-type library --build-type ${{ parameters.SDKType }} --avi --sr
+                    displayName: Update pom files for incremented versions
+
+                  - template: /eng/common/pipelines/templates/steps/create-pull-request.yml
+                    parameters:
+                      PRBranchName: increment-package-version-${{ parameters.ServiceDirectory }}-$(Build.BuildId)
+                      CommitMsg: "Increment package versions for ${{ parameters.ServiceDirectory }} releases"
+                      PRTitle: "Increment versions for ${{ parameters.ServiceDirectory }} releases"
+                      PRLabels: "auto-merge"
+                      CloseAfterOpenForTesting: '${{ parameters.TestPipeline }}'
+
+        # JRS - This needs to be updated with changes to batch up Docs.MS publishing otherwise
```

Comment:
left over todo?

Reply:
Yeah, that todo is done. The next commit will have this comment removed.